understanding/reflow.html

```diff
@@ -44,109 +44,108 @@ <h1>達成基準 1.4.10: リフローを理解する</h1>
                
-               <p>For organisations which are using legacy systems or are not able to update their layout methods for some reason, an alternative conforming version could be a mobile site which has a fixed 320px wide layout. The user should be able to find that version from the default website.</p>
+               <p>旧来のシステムを使用しているか、何らかの理由でレイアウト方法を更新できない構成の場合、代替の適合バージョンは、320px 幅の固定レイアウトを持つモバイルサイトである。利用者はデフォルトのWebサイトからそのバージョンを見つけることができるようにすべきである。</p>
                	
             </section>
             <section id="avoiding-scrolling-in-horizontally-and-vertically-written-languages">
                
-               <h3>Avoiding scrolling in horizontally and vertically written languages</h3>
+               <h3>横書き及び縦書きの言語でのスクロールの回避</h3>
                
-               <p>The success Criterion applies to both horizontally and verticaly written languages. Zooming the page for horizontally written languages where pages scroll vertically by default (e.g. English) should not require horizontal scrolling. Zooming the page for vertically written languages which scroll horizontally by default should not require vertical scrolling.</p>
+               <p>達成基準は、横書きと縦書きの両方の言語に適用される。英語などの横書きの言語にとって、デフォルトで縦方向にスクロールするページをズームする場合、横方向のスクロールを発生させるべきではない。縦書きの言語にとって、デフォルトで横方向にスクロールするページをズームする場合、縦方向のスクロールを発生させるべきではない。</p>
                	
             </section>
             <section id="the-relation-of-reflow-to-the-success-criterion-1.4.4-resize-text">
                
-               <h3>The relation of Reflow to the Success Criterion 1.4.4 Resize Text</h3>
+               <h3>リフローと達成基準 1.4.4 テキストのサイズ変更の関係</h3>
                
-               <p>The focus of the Reflow Success Criterion is to enable users to zoom in without having to scroll in two directions. <a href="resize-text.html" class="sc">Success Criterion 1.4.4 Resize Text</a> also applies, so it should be possible to increase the size of all text to at least 200% while simultaneously meeting the reflow requirement. If text is reduced in size when people zoom in (or for small-screen usage), it should still be possible to get to 200% enlargement. For example, if text is set at 20px when the window is 1280px wide, at 200% zoom it should be at least 20px (so 200% of the default size), but at 400% zoom it could be set to 10px (therefore still 200% of the default 100% view). It is not required to achieve 200% enlargement at every breakpoint, but it should be possible to get 200% enlargement in some way.</p>
+               <p>リフローの達成基準の焦点は、利用者が2 方向にスクロールしなくてもズームインできるようにすることである。<a href="resize-text.html" class="sc">達成基準 1.4.4 テキストのサイズ変更</a>も適用することで、リフローの要件を満たしながら、すべてのテキストのサイズを少なくとも200％ に拡大することができる。利用者がズームインしたとき（又は小さなスクリーンで利用しているとき）にテキストのサイズが縮小された場合でも、200％ の拡大率を得ることは可能である。たとえば、ウィンドウの幅が1280px の場合にテキストが20px で設定されている場合、200％ ズームでは最低20px（デフォルトサイズの200％）になるが、400％ ズームでは（デフォルトの100％ 表示の200％ なので）10px に設定できる。すべてのブレークポイントで200％ の拡大を可能にする必要はないが、何かしらの方法で200％ の拡大可能にすべきである。</p>
```

Comment:
すみません。あと出して申し訳ないのですが、ここ改めて見直してみたのですが、原文がどういったことをいいたいのかイマイチわかりませんでした。

>For example, if text is set at 20px when the window is 1280px wide, at 200% zoom it should be at least 20px (so 200% of the default size), but at 400% zoom it could be set to 10px (therefore still 200% of the default 100% view).

を

>例えば、ウィンドウの幅が1280px のときにテキストが 20px に設定されている場合、200 ％ズームでは最低 20px (デフォルトサイズの 200 ％) になるが、 400 ％ズームでは (デフォルトの 100 ％表示の まだ 200 ％であるので) 10px に設定できる

と（レビューを受けて）訳しましたが私の理解不足なのかもしれませんが、20pxで設定されたテキストを200％に拡大したら40pxになるのではないのでしょうか。400％の話やご指摘いただいたカッコの箇所も原文がどういったことを言いたいのかわからず、、どう訳すべきか迷っています。
ご意見ありましたら伺いたく思います。。

Reply:
> あと出して申し訳ない

全然OKです…！遠藤さんの疑問はごもっともだと思います。しかし、[該当箇所の最新の原文](https://w3c.github.io/wcag/understanding/reflow.html#the-relation-of-reflow-to-the-success-criterion-1.4.4-resize-text)が書き換わってますので、この文章には深く突っ込まず、とりあえず訳注で対応するのはどうでしょうか（ここで頑張っても次の翻訳の更新で翻訳も変わりますので）。

訳注はこんな感じでどうでしょうか？

```
<div class="note">
<div role="heading" class="note-title marker" aria-level="4">訳注</div>
<div>
<p>
「リフローと達成基準 1.4.4 テキストのサイズ変更の関係」の説明には不明瞭な箇所があるが、この箇所の翻訳は原文に忠実な形で行った。なお、最新の原文の該当箇所は大幅に加筆されている。Understanding WCAG 2.1 <a href="https://w3c.github.io/wcag/understanding/reflow.html#the-relation-of-reflow-to-the-success-criterion-1.4.4-resize-text">The relation of Reflow to the Success Criterion 1.4.4 Resize Text</a> もあわせて参照のこと。
</p>
</div>
</div>

```